23 Installing EF Core to a computer

arynhuck opened the pull request:
This would close issue #23

arynhuck committed:
Content added

We've been working on OneDrive to have a more collaborative platform to
adjust our content and now this is a content dump.

This includes updates to the README and TOC to include references to
this page.

helmoski commented on content/efcore/getting-started/installing-ef-core.md:
````diff
@@ -0,0 +1,58 @@
+If you haven't already, you will first need to install .NET Core on your machine. Go to the [.NET Core Page](https://www.microsoft.com/net/core) of the Microsoft website, and follow the installation instructions for your platform of choice. You can ensure .NET Core is installed by running the following command: 
+ 
+``` 
+dotnet –version 
````
should be `--version`

arynhuck replied:
done

helmoski commented on content/efcore/getting-started/installing-ef-core.md:
```diff
@@ -0,0 +1,58 @@
+"tools": { 
+    "Microsoft.EntityFrameworkCore.Tools": "1.0.0-preview2-final" 
+} 
+ 
```
Remove extra newline

arynhuck replied:
done

helmoski commented on content/efcore/getting-started/toc.md:
```diff
@@ -1 +1,2 @@
-- [Intro to EF Core](what-is-ef-core.md)
\ No newline at end of file
+- [Intro to EF Core](what-is-ef-core.md)
```
I should have caught this in PR #20, but the url should be `intro-to-ef-core.md`

arynhuck replied:
done

helmoski commented on content/efcore/getting-started/README.md:
```diff
@@ -12,4 +12,5 @@ This tutorial is aimed at people who are learning EF Core for the first time. Mo
 This tutorial will cover the following topics:
-- [Intro to EF Core](what-is-ef-core.md)
\ No newline at end of file
+- [Intro to EF Core](what-is-ef-core.md)
```
I should have caught this in PR #20, but the url should be `intro-to-ef-core.md`

arynhuck replied:
Yep. That is a thing we did not catch. 😐  _panics_

helmoski commented on content/efcore/getting-started/installing-ef-core.md:
```diff
@@ -0,0 +1,58 @@
+# Installing EF Core  
+ 
+In this lesson, you will learn how to install EF Core on your machine. You do not need to complete these steps to continue with the rest of the tutorial. 
```
Just thought of this: We should probably state the reason they don't need to finish the steps in this lesson (The code examples in the tutorial can be run in-browser)

arynhuck replied:
done

arynhuck committed:
Small content change, fixed broken file name links

Just why do users get to skip this lesson? We didn't really tell them...
But now we have!

Apparently we missed that the filenames in the TOC and the README for
the Intro to EF Core page were wrong (we renamed the file).

arynhuck commented:
Adjustments made! :shipit: ?

helmoski commented on content/efcore/getting-started/installing-ef-core.md:
```diff
@@ -0,0 +1,57 @@
+  
+## Add EF Core to Project 
+ 
+ 
```
Remove extra newline

arynhuck committed:
Update installing-ef-core.md

helmoski commented:
:shipit:
